hazelcast/src/test/java/com/hazelcast/spi/impl/operationservice/impl/CompletableFutureTestUtil.java

```diff
@@ -55,7 +55,7 @@ public void run() {
             public void run() {
                 throw new ExpectedRuntimeException();
             }
-        }) : invokeAsync(instance, new SlowOperation(3000));
+        }) : invokeAsync(instance, new SlowOperation(1000));
```

Comment:
on my local this reduced test execution time for `BiCompletionStageTest` from 30sec to 12sec. On Jenkins PR builder improvement was from 2m 5sec (PR builder 2802) to 45sec (2813).